cli/flox/src/utils/search.rs

```diff
@@ -7,20 +6,15 @@ use crossterm::style::Stylize;
 use crossterm::tty::IsTty;
 use flox_rust_sdk::models::search::{SearchResult, SearchResults};
 
-pub const SEARCH_INPUT_SEPARATOR: &'_ str = ":";
 pub const DEFAULT_DESCRIPTION: &'_ str = "<no description provided>";
 
 /// An intermediate representation of a search result used for rendering
 #[derive(Debug, PartialEq, Clone)]
 pub struct DisplayItem {
-    /// The input that the package came from
-    input: String,
-    /// The attribute path of the package, excluding subtree and system
-    rel_path: Vec<String>,
+    /// The package path of the package, including catalog name
+    pkg_path: String,
```

Comment:
nonblocking suggestion: related to the other thread, if `nixpkgs` gets included in `pkg_path`, then I think we'll want this to be `catalog` and `attr_path` rather than `pkg_path` so we don't have to strip `nixpkgs/` when printing

Reply:
`pkg_path` is by definition the field we want to print and it should never include `nixpkgs`. The problem is just that the `/packages` endpoint doesn't return `pkg_path` yet.

Reply:
Gotcha got thrown off by `pkg-path = "<catalog>/<attr-path>"` but I see you meant in that case catalog would have to be null for nixpkgs